docs/src/tutorials/algorithms/benders_decomposition.md

```diff
@@ -0,0 +1,8 @@
+```@meta
+EditURL = "benders_decomposition.jl"
+```
+
+# [Benders decomposition](@id benders_decomposition_classical)
+
+This page is a placeholder that appears only if the documentation is built from
+a fork.
```

Comment:
Call says split this into two tutorials, with and without callbacks to minimize the bits that Gurobi uses.